### DISCUSSION TITLE
CasaOS Text in Terminal Unreadable

### DISCUSSION THREAD

**Discussion by 1artist** [IceWhaleTech/CasaOS#502] [General]
Hi All,
When I SSH connect to my RasPi Lite 64bit OS using the inbuilt CasaOS terminal  window what I see in my CasaOS terminal display  is a bunch of unreadable text that looks like it is all vertically striped in many colors. Is this a known bug?  If no, how do I fix this problem? Any advice welcome. 

System Info:
OS: Linux Mint
CasaOS v 0.3.5.1
Plenty of Disk space
Browser: FireFox

**Comment by ghost** [comment]
Hi 1artist,
This is caused by canvas settings in my experience. If I enable 'resist fingerprinting' on Firefox, I get the colours instead of text. I expect a canvas spoofing add-on may do the same.
